recipe/meta.yaml

```diff
@@ -25,10 +25,11 @@ requirements:
   build:
     - python
     - setuptools
+    - pytest-runner
 
   run:
     - python
-    - astroid >=1.4.5,<1.5.0
+    - astroid >=1.5.1,<1.6.0
     - backports.functools_lru_cache  # [py27]
     - configparser                   # [py27]
```

Comment:
Appears they also use [`singledispatch` for `py2k`]( https://github.com/PyCQA/pylint/blob/pylint-1.7.1/pylint/__pkginfo__.py#L48 ). Probably should make these selectors `py2k` too.